### PR TITLE
Rm lodash

### PR DESCRIPTION
**Pull request by anithri**
Lodash is pretty heavy, so I removed lodash in favor of ES6 & ES7 native equivilents.

I did not remove it from `src/__tests__/metaverse.spec.js` yet.  I need to look at the file with fresh eyes cause I can't figure out why it's doing certain things.

### CODE REVIEW

File: src/context.js [hunk 1]
```diff
@@ -17,20 +16,23 @@ const helpers = {
...
-    v => helpers.capitalize(v)
-  )
+  Object.entries(locals).reduce((hsh, [key, value]) => {
+    hsh[key] = value
+
+    if (localsToCapitalize.includes(key))
+      hsh[helpers.capitalize(key)] = helpers.capitalize(value)
+
+    return hsh
+  }, {})
```
Comment (jondot) [inline src/context.js]: I'm wondering, if here you can create some kind of higher order function locally, like `filter` in this portion of the code:

https://github.com/jondot/hygen/blob/b2925f1eafa63bc73175ecba7122b99a52f790ec/src/render.js#L16

in this case the `mapValues`, `pick`, and `mapKeys` kind of contributed to readability.

File: src/help.js [hunk 2]
```diff
@@ -41,11 +36,12 @@ const printHelp = (templates: string, logger: Logger) => {
...
       See http://hygen.io for more.
       
-      `
+      `,
     )
     return
   }
-  L.each(availableActions(templates), (v, k) => {
+  Object.entries(availableActions(templates)).forEach(([k, v]) => {
+    console.log('ACTIONS', k, v)
```
Comment (jondot) [inline src/help.js]: Runaway console.log?

**Comment by jondot** [comment]
What a monumental effort! Thanks!
I'm excited to see how this affects start up time

**Comment by anithri** [comment]
I think that's it. 

Let me know if you don't like the refactors

**Pull request merged**